javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -248,9 +248,25 @@ public static boolean isIntegral(TypeMirror type) {
         }
     }
 
+    /**
+     * Returns true iff the argument is a boxed floating point type.
+     *
+     * @param type type mirror
```

Comment:
I would prefer something more descriptive than the type, although "type to test" is also not much better.